packages/twenty-front/src/modules/auth/hooks/__tests__/useAuth.test.tsx

```diff
@@ -13,12 +14,14 @@ import { supportChatState } from '@/client-config/states/supportChatState';
 import { workspaceAuthProvidersState } from '@/workspace/states/workspaceAuthProvidersState';
 
 import { isMultiWorkspaceEnabledState } from '@/client-config/states/isMultiWorkspaceEnabledState';
-import { email, mocks, password, results, token } from '../__mocks__/useAuth';
 import { renderHook } from '@testing-library/react';
+import { email, mocks, password, results, token } from '../__mocks__/useAuth';
 
 const Wrapper = ({ children }: { children: ReactNode }) => (
   <MockedProvider mocks={mocks} addTypename={false}>
-    <RecoilRoot>{children}</RecoilRoot>
+    <RecoilRoot>
+      <MemoryRouter>{children}</MemoryRouter>
```

Comment:
Why do you need MemoryRouter?

Reply:
The `MemoryRouter` is required here because we recently introduced `useSearchParams` in the `useAuth` hook. Like other React Router hooks, `useSearchParams` needs to be wrapped in a Router context to function properly. Without `MemoryRouter`, the test fails with the error "useLocation() may be used only in the context of a <Router> component" since `useSearchParams` internally depends on `useLocation`.